theia-deb-build-docker/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "theia",
+    "version": "0.8.0",
```

Comment:
I would suggest having a version for this demo app that's distinct from the Theia platform version. Else it will need to be updated often or will not reflect reality. Specially since this is then used to name the .deb package, and so is more visible. Suggestion 0.0.1 :) The name as well could be something like "debian-packaging-example" or something similar.

Reply:
The version is now hardcoded to be `version=0.0.1`. The node-deb module only uses the `name` field in `package.json` to name the final `.deb` file.

For example, if `name=theia-test` and `version=0.0.1`, the final .deb file would be called `theia-test_0.0.1_all.deb`. The filename does not affect calling `theia` from the commandline once the package has been installed by apt.

Reply:
Thanks for the explanation. 

How about using `theia-example` as name, then? It's a minor detail, but we can see that there is already quite a bit of confusion between `Theia` the platform and calling specific applications the same.  e.g. think of calling an IDE developed on the Eclipse platform "eclipse"  ;)

Reply:
Changed `name=theia-example`.